### PR TITLE
Add reproduce frontend test execution order and update windows installation method

### PR DESCRIPTION
**Pull request by gp201**
This PR does the following:

- Adds information on how to reproduce frontend test execution order.
- Updates windows installation method.

### CODE REVIEW

**Review by U8NWXD — CHANGES_REQUESTED** [review]
Please update the tables of contents. Otherwise LGTM

**Comment by gp201** [comment]
@U8NWXD PTAL

File: Debug-frontend-tests.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 * [Find tests](#find-tests)
 * [Isolate tests](#isolate-tests)
 * [Verbose mode](#verbose-mode)
+* [Downloading the combined-tests.spec.js file](#downloading-the-combined-tests.spec.js-file)
```
Comment (U8NWXD) [inline Debug-frontend-tests.md]: This link doesn't work. You can try it out here: https://github.com/oppia/oppia-web-developer-docs/blob/2e86b515a9d7db2b1a9eafbe13ed3f569b4cb7c8/Debug-frontend-tests.md

File: Installing-Oppia-(Windows;-Python-3).md [hunk 2]
```diff
@@ -160,15 +160,15 @@ You will be asked about the setup for the panel. Just click on default config.
 
 ![image](https://user-images.githubusercontent.com/23002114/94156539-d24fff00-fe9d-11ea-8f7a-039db7a7ce51.png)
 
-## Run the E2E tests
+## Step 9: Run Oppia locally
```
Comment (U8NWXD) [inline Installing-Oppia-(Windows;-Python-3).md:164]: This file's table of contents needs to be updated too

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Debug-frontend-tests.md (hunk 1 above)
Comment (gp201) [reply Debug-frontend-tests.md]: Fixed

File: Installing-Oppia-(Windows;-Python-3).md (hunk 2 above)
Comment (gp201) [reply Installing-Oppia-(Windows;-Python-3).md:164]: Done

**Comment by gp201** [comment]
@U8NWXD PTAL

File: Installing-Oppia-(Windows;-Python-3).md [hunk 3]
```diff
@@ -10,6 +10,7 @@
     * [Edit desktop environment config](#edit-desktop-environment-config)
   * [Step 7: Start the rdp server](#step-7-start-the-rdp-server)
   * [Step 8: Connect to the server using the Windows RDP client](#step-8-connect-to-the-server-using-the-windows-rdp-client)
+  * [Step 9: Run Oppia locally](#step-9-run-oppia-locally)
   * [Run the e2e tests](#run-the-e2e-tests)
```
Comment (U8NWXD) [inline Installing-Oppia-(Windows;-Python-3).md]: This TOC entry is wrong still

**Review by U8NWXD — CHANGES_REQUESTED** [review]

File: Installing-Oppia-(Windows;-Python-3).md (hunk 3 above)
Comment (gp201) [reply Installing-Oppia-(Windows;-Python-3).md]: Sorry about that. Fixed it.

**Review by U8NWXD — APPROVED** [review]

**Pull request merged**